clustered/clustered-dist/build.gradle

```diff
@@ -93,7 +86,7 @@ jar {
     'Bundle-SymbolicName': 'org.ehcache.clustered',
     'Export-Package': '!com.tc.*, !com.terracotta.*, !org.terracotta.*, !org.ehcache.*.internal.*, !sun.misc, ' +
       'org.ehcache.clustered.client.*, org.ehcache.clustered.common.*',
-    'Import-Package': '!sun.misc.*, org.ehcache.xml.*;resolution:=optional, jdk.jfr.*;resolution:=optional, *'
+    'Import-Package': '!sun.misc.*, org.ehcache.xml.*;resolution:=optional, jdk.jfr.*;resolution:=optional, com.fasterxml.jackson.annotation;resolution:=optional, *'
```

Comment:
Would be good to explore if this can be a straight exclusion rather than an optional import.

Reply:
yes, should be solved by https://github.com/Terracotta-OSS/terracotta-platform/pull/647

Reply:
Put the maven provided scope so that `jackson-annotations` does not come transitively (5.7.4-pre10), but I still need the addition of the line otherwise the test fails.

Reply:
Right - but you should be able to use a straight 'exclusion'. In fact to truly assert this isn't needed you should be able to apply a transitive dependency exclusion at the point where you depend on `common-structures` as well.

Reply:
oh! yes true!